tests/test_imports.py

```diff
@@ -6,41 +6,61 @@
 
 
 @fixture
+
+#TODO NOW: update to use real import.yaml file. Unclear if this is the right thing to do based on how the tests are set up. 
```

Comment:
Has this been done?

Reply:
No, if we do this now it won't test the MAGs function so I have left this as a future TODO.